velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -484,6 +484,72 @@ struct ArrayCumSumFunction {
   }
 };
 
+template <typename TExecCtx, typename P, typename S>
```

Comment:
Simple function framework support decimals, do we need a separate struct for decimals? Can we use the existing implementation for decimals, since the int64 and int128 physical types are already supported?